index.html

```diff
@@ -413,11 +413,119 @@ <h2>General comments: Pros and Cons</h2>
+
+                    <section>
+                        <h2>Turtle</h2>
+                        <pre class=nohighlight>
+                                [] ex:example [
+                                    rdf:value "<bdo dir="ltr">&#x05E4;&#x05E2;&#x05D9;&#x05DC;&#x05D5;&#x05EA; &#x05D4;&#x05D1;&#x05D9;&#x05E0;&#x05D0;&#x05D5;&#x05DD;, W3C</bdo>" ;
+                                    rdf:language "he" ; 
+                                    rdf:direction "rtl" ;
+                                ] .
+                        </pre>
+                                    
+                        <p>Note that, compared to <a href="#langstring-example">previous examples</a> the text does not appear directly as the object of a triple, but is encapsulated in a separate (blank) node that “represents” the text with all attributes attached.</p>
+                    </section>
+
+                    <section>
+                        <h2>JSON-LD</h2>
+                        <p>Interestingly, due to the specificities of the JSON-LD syntax, the serialization of compound literals is extremely straightforward. Indeed, a syntax of the form:</p>
+
+                        <pre class=nohighlight>
+                            "ex:example" : {
+                                "@value" : "<bdo dir="ltr">&#x05E4;&#x05E2;&#x05D9;&#x05DC;&#x05D5;&#x05EA; &#x05D4;&#x05D1;&#x05D9;&#x05E0;&#x05D0;&#x05D5;&#x05DD;, W3C</bdo>"
+                                "@language" : "he",
+                                "@direction" : "rtl"
+                            }
+                        </pre>
+            
+                        <p>may be mapped onto compound literals directly.</p>
+                    </section>
+                </section>
+                <section>
+                    <h2>Pros and Cons</h2>
+                    <p>
+                        The major advantage of this approach is that it is fully transparent to current RDF deployments, and affects only application layers that do display the literal values for humans. They also work for parsers “out of the box”. As an extra bonus (as mentioned <a href='https://github.com/w3c/EasierRDF/issues/22'>“Language Tagged Strings”</a> discussion) it may actually be easier to search such compound literals via, say, SPARQL then using the current RDF approach.
+                    </p>
+
+                    <p>
+                        The obvious disadvantage is that it duplicates a mechanism for expressing string with the language tag <em>only</em>: one can use the core <code>langString</code> datatype <em>or</em> the compound literals. The relationship between the two forms should be specified.
+                    </p>
+
+                    <p>
+                        (It would be possible to make a “mix” whereby, instead of using <code>rdf:language</code>, one would use a <code>rdf:langString</code> as an object for <code>rdf:value</code>.)
```

Comment:
Yuk :-/ I am not sure I want to hint at that solution... IMO, it has the worse of both worlds: there is still an indirection to follow to "find" the actual string, without the benefit of cleanly separating its metadata in distinct triples... Worth, the metadata are heterogeneous in that respect.

I suggest we remove this parenthesis.

Reply:
I am not saying I like it... :-)

I am fine removing it, but you can bet someone will raise this!